docs/porting/connectivity/Layer3IP.md

```diff
@@ -0,0 +1,131 @@
+<h2 id="ethernet-port">IP Layer3 (L3IP) drivers</h2>
```

Comment:
Please choose another unique heading ID because "ethernet-port" is already used here: https://os.mbed.com/docs/mbed-os/v5.11/porting/ethernet-port.html

Reply:
@tymoteuszblochmobica  use `id="l3-porting"` here.
The `id` field is used by the documentation engine to generate the .html file name.

Reply:
done